src/murmur/MurmurIce.cpp

```diff
@@ -40,15 +40,32 @@ void IceStop() {
 	mi = NULL;
 }
 
+/// iceRemoveNul removes all NUL bytes from |s|.
+static std::string iceRemoveNul(std::string s) {
+	s.erase(std::find(s.begin(), s.end(), '\0'));
```

Comment:
This will only remove the first occurrence?

[Find returns an iterator (case 1)](http://en.cppreference.com/w/cpp/algorithm/find), [erase erases the character at that position (case 2)](http://en.cppreference.com/w/cpp/string/basic_string/erase).